src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -981,6 +991,8 @@ export class HaConfigDevicePage extends LitElement {
     if (configurationUrl) {
       deviceActions.push({
         href: configurationUrl,
+        rel: "noreferrer",
+        target: configurationUrlIsHomeAssistant ? "_self" : "_blank",
```

Comment:
```suggestion
        target: configurationUrlIsHomeAssistant ? undefined : "_blank",
```

Reply:
We can automatically add `noreferrer` if target is defined and remove the option.